epicsdbbuilder/recordbase.py

```diff
@@ -227,13 +231,16 @@ def __ValidateField(self, fieldname, value):
             self._validate.ValidFieldValue(fieldname, str(value))
 
     # Field formatting
-    def __FormatFieldForDb(self, fieldname, value):
+    def __FormatFieldForDb(self, fieldname, value, oneline=False):
```

Comment:
Instead of oneline, could we keep the current prettyprinting, but pass in `lineprefix` which defaults to `''` but could also be `feature_flag_macro`?

Reply:
Done